Knossos.NET/Classes/Knossos.cs

```diff
@@ -1162,6 +1162,40 @@ public static async void PlayMod(Mod mod, FsoExecType fsoExecType, bool standalo
 
             Log.Add(Log.LogSeverity.Information, "Knossos.PlayMod()", "Used cmdLine : " + cmdline);
 
+            if (MainWindow.instance != null && globalSettings.warnNewSettingsSystem)
+            {
+                try
+                {
+                    var fsoVersion = new SemanticVersion(fsoBuild.version);
+                    var newSettingsVersion = new SemanticVersion("24.2.0-RC");
+                    if(fsoVersion >= newSettingsVersion)
+                    {
+                        await Dispatcher.UIThread.InvokeAsync(async () => {
+                            try
+                            {
+                                var res = await MessageBox.Show(MainWindow.instance, "Starting from FSO version 24.2.0 a new in-game settings system was implemented,\n" +
```

Comment:
```
"Starting from FSO version 24.2.0 a new in-game settings system was implemented.\n" +
 "Launcher settings, such as video, audio or input, and Command Line arguments of those categories no longer have an effect by default.\n" +
 "Settings must be now changed in-game by going to the 'Options' menu, where you will see additional methods for accessing and setting these new in-game options.", "New FSO settings system",
 ```
 
 I've suggested updated wording for this section. 1) Technically mods can set a table setting to disable this new in-game options, which FotG currently does, so not all mods may have it on by default. Also, the player can turn this off with a Command Line flag, but it is generally not recommended for common players and more used for use devs. 2) Not every mod that uses the new in-game settings options requires you to press f3, some just have them shown. FS2 retail mod has text on the menu screen telling the player what to press, so that's covered in-game.